syntax = "proto3";
package SeedInfoServer;
message seedInfo {
    map<string,string> seedList = 1;
}
